plugins/services/src/js/reducers/serviceForm/__tests__/LocalVolumes-test.js

```diff
@@ -4,13 +4,24 @@ const Transaction = require('../../../../../../../src/js/structs/Transaction');
 const {ADD_ITEM, REMOVE_ITEM, SET} =
   require('../../../../../../../src/js/constants/TransactionTypes');
 
+const LOCAL_VOLUME_DEFAULT_SIZE =
+  require('../../../constants/LocalVolumes').DEFAULT_SIZE;
+
 describe('LocalVolumes', function () {
   describe('#FormReducer', function () {
     it('should return an Array with one item', function () {
       const batch = new Batch()
         .add(new Transaction(['localVolumes'], 0, ADD_ITEM))
         .add(new Transaction(['localVolumes', 0, 'type'], 'PERSISTENT'));
-      expect(batch.reduce(LocalVolumes.FormReducer, [])).toEqual([{size: null, containerPath: null, mode: 'RW', type: 'PERSISTENT'}]);
+      expect(batch.reduce(LocalVolumes.FormReducer, []))
+      .toEqual([
```

Comment:
:( Our linter is not catching this. I think your editor is configured with a weird setting